src/Umbraco.Cms.Integrations.SEO.Semrush/Api/Management/Controllers/GetCurrentContentPropertiesController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Asp.Versioning;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.Integrations.SEO.Semrush.Configuration;
+using Umbraco.Cms.Integrations.SEO.Semrush.Models.Dtos;
+using Umbraco.Cms.Integrations.SEO.Semrush.Services;
+
+namespace Umbraco.Cms.Integrations.SEO.Semrush.Api.Management.Controllers
+{
+    [ApiVersion("1.0")]
+    [ApiExplorerSettings(GroupName = Constants.ManagementApi.SemrushGroupName)]
+    public class GetCurrentContentPropertiesController : SemrushControllerBase
+    {
+        private readonly IContentService _contentService;
+        private readonly IContentTypeService _contentTypeService;
+        public GetCurrentContentPropertiesController(IOptions<SemrushSettings> options,
+            IWebHostEnvironment webHostEnvironment,
+            ISemrushTokenService semrushTokenService,
+            ICacheHelper cacheHelper,
+            TokenBuilder tokenBuilder,
+            SemrushComposer.AuthorizationImplementationFactory authorizationImplementationFactory,
+            IContentService contentService,
+            IContentTypeService contentTypeService,
+            IHttpClientFactory httpClientFactory) : base(options, webHostEnvironment, semrushTokenService, cacheHelper, tokenBuilder, authorizationImplementationFactory, httpClientFactory)
+        {
+            _contentService = contentService;
+            _contentTypeService = contentTypeService;
+        }
+
+        [HttpGet("content-properties")]
+        [ProducesResponseType(typeof(List<ContentPropertyDto>), StatusCodes.Status200OK)]
+        public async Task<IActionResult> GetCurrentContentProperties(string contentId)
+        {
+            var propertyList = new List<ContentPropertyDto>();
+            var currentContent = _contentService.GetById(Guid.Parse(contentId));
```

Comment:
I had to use both `IContentService` and `IContentTypeService` here because `IContentTypeService` can get properties of a content with its group, but cannot get their values. That's why I also used `IContentService` to get their corresponding values.

Reply:
Would suggest `contentId` can be passed as a GUID, which will save parsing it for the call to get it by Id.

Need to check here too if this logic works with compositions - e.g. content type A as a property, and composition of content type B that also has a property.  I think you'll only get the property from content type A here.